test/conformance/kuttl/generate/clusterpolicy/standard/clone/nosync/cpol-clone-nosync-delete-policy/singlerule.yaml

```diff
@@ -0,0 +1,21 @@
+apiVersion: kyverno.io/v2beta1
+kind: ClusterPolicy
+metadata:
+  name: cpol-nosync-clone
+spec:
```

Comment:
singlerule.yaml not used in `cpol-clone-nosync-delete-policy` test, we can remove.